src/simulator/src/verilogHelpers.ts

```diff
@@ -20,7 +20,7 @@ export function sanitizeLabel(name) {
     return temp
 }
 
-export function generateNodeName(node, currentCount, totalCount) {
+export function generateNodeName(node: { verilogLabel: any; parent: { verilogLabel: any }; label: any }, currentCount: string | number, totalCount: number) {
```

Comment:
- Please prefer interface that will help in future to extend.
- Please prefer using correct type or `unknown` over `any`